src/CSS.php

```diff
@@ -66,6 +66,29 @@ public function setImportExtensions(array $extensions)
     }
 
     /**
+     * Move any import statements to the top
+     *
+     * @param $content string Nearly finished CSS content
+     *
+     * @return string
+     */
+    protected function moveImportsToTop($content) {
+
+        if (preg_match_all('#@import[^;]+;#', $content, $matches)) {
```

Comment:
Can you use `/` as a delimiter? Pet peave of mine - I wish everyone standardized on `/` (and that's what I already use elsewhere throughout the file) :)